requirements_qa.txt

```diff
@@ -1 +1,2 @@
+beautifulsoup4==4.9.3
```

Comment:
beautiful soup is not the CI dependency, right? the requirements.txt file is the right place to enumerate all dependencies